src/main/java/picard/vcf/GatherVcfs.java

```diff
@@ -45,9 +48,28 @@
 
     @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Output VCF file.")
     public File OUTPUT;
+    
+    @Argument(doc = "Comment(s) to include in the merged output file's header.", optional = true, shortName = "CO")
+    public List<String>  COMMENT = new ArrayList<>();
+    
+    @Argument(doc = "The variants in INPUT come from non overlapping regions "
```

Comment:
It would be clearer here if you start with what the option is doing and then explain the reason:
```suggestion
    @Argument(doc = "If 'true' the program will reorder INPUT according "
            + "to the genomic location of the first variant in each file. "
            + "this is useful since the order of variants in each file in INPUT come from non overlapping regions "
            + " but the order of the files in INPUT is untrusted.",
    optional = true, shortName = "RI")
```